community.md

```diff
@@ -0,0 +1,105 @@
+- <a href="https://github.com/bolav/fuse-homescreen">UX action to go to the home screen on Android by <strong>bolav</strong></a>
+- <a href="https://github.com/Duckers/Fuse.MaterialDesign">Cross-platform, open source implementation of Google&#39;s material design in Fuse by <strong>Duckers</strong></a>
+- <a href="https://github.com/torial/fuse-community">Onetimepassword and other integrations by <strong>torial</strong></a>
+- <a href="https://github.com/bolav/fuse-example-using-socketio">Socket.io by <strong>bolav</strong></a>
+- <a href="https://github.com/bolav/fuse-sqlite">Use SQLite database from JS in Fuse by <strong>bolav</strong></a>
+- <a href="https://github.com/bolav/fuse-example-using-syncano">Syncano backend example by <strong>bolav</strong></a>
+- <a href="https://github.com/bolav/fuse-testfairy">TestFairy integration by <strong>bolav</strong></a>
+- <a href="https://github.com/bolav/fuse-touchid">Touch ID integration (iOS) by <strong>bolav</strong></a>
+- <a href="https://github.com/fusetools/UnoBashCompletion">Simple bash auto-completion for Uno CLI by <strong>Fuse</strong></a>
+- <a href="https://github.com/Hazealign/vscode-fuse">VSCode plugin by <strong>hazealign</strong></a>
+- <a href="https://marketplace.visualstudio.com/items?itemName=naumovs.vscode-fuse-syntax">Syntax highlighting in Visual Studio Code by <strong>Sergii Naumov</strong></a>
+- <a href="https://github.com/bolav/fuse-whilevisibleinscrollview">WhileVisibleInScrollView UX trigger by <strong>bolav</strong></a>
+- <a href="https://github.com/bolav/fuse-youtube">YouTube integration by <strong>bolav</strong></a>
+- <a href="https://github.com/heavenchains/FuseShareSheet">Native share sheets by <strong>Heavenchains</strong></a>
+- <a href="https://github.com/bstrr/vscode-fuse-log">Get Fuse log messages in VS Code by <strong>bstrr</strong></a>
+- <a href="https://github.com/benny2871/fuse-backgroundservice">Long running services for Android by <strong>Benny2871</strong></a>
+- <a href="https://github.com/blackrider401/Fuse_Android_Toast">Use the native Android Toast view in Fuse by <strong>blackrider401</strong></a>
+- <a href="https://github.com/jesusmartinoza/Fuse.SocialShare">Library to use social share options in Fuse by <strong>jesusmartinoza</strong></a>
+- <a href="https://github.com/akalyan/fuse-cordova-bridge">Project that allows cordova plugins to be used in Fuse apps by <strong>akalyan</strong></a>
+- <a href="https://github.com/jmoleiro/fuse-dfp">Implementation of DoubleClick for Publishers SDK in Fuse by <strong>jmoleiro</strong></a>
+- <a href="https://github.com/bkono/listenup">Example of Snowboy hotword detection in Fuse by <strong>bkono</strong></a>
+- <a href="https://github.com/IsaacSomething/fuse-tools-icon-sets">A collection of files with icon code mappings to quickly add icons to your fuse tools project by <strong>IsaacSomething</strong></a>
```

Comment:
Isn't this kind of just a repetition of [this](https://fuseopen.com/docs/packages.html)?

Reply:
I think the docs one should just link to the website one

Reply:
I'm having a hard time convincing myself which is better. It kinda feels a bit like it doesn't matter... But perhaps the website is a bit easier to work with?

Reply:
I believe the website version is more noticeable cos when you're in the docs, it can be a bit overwhelming and you can miss it entirely as the initial user flow is the Installation and quickstart > intro/tutorial > anything, having both links isn't bad, just bad to maintain 2 lists. Maybe should add a paragraph on how to typically use a package in the website version until we dust them all off and npm them 😄